### PR TITLE
[4.1] Action Logs improved search

### PR DESCRIPTION
**Pull request by brianteeman**
This simple PR changes the search in the action logs to include the message field (aka the Action Column) so that now you can find every action that touched a specific article etc.

As this includes the username we can simply change the query from username to message

To test make sure you have more than 1 user that has performed actions on the site. Then make sure that the search finds the results you expect **including** testing the searches with the filters applied

Pull Request for Issue #36013 

cc @coolcat-creations is this what you are looking for

### CODE REVIEW

File: administrator/components/com_actionlogs/src/Model/ActionlogsModel.php [hunk 1]
```diff
@@ -166,8 +166,8 @@ protected function getListQuery()
 			else
 			{
 				$search = '%' . $search . '%';
-				$query->where($db->quoteName('u.username') . ' LIKE :username')
-					->bind(':username', $search);
+				$query->where($db->quoteName('a.message') . ' Like :message')
```
Comment (Quy) [inline administrator/components/com_actionlogs/src/Model/ActionlogsModel.php]:
```suggestion
				$query->where($db->quoteName('a.message') . ' LIKE :message')
```

**Comment by coolcat-creations** [comment]
I have tested this item :white_check_mark: successfully on cddacdee5434c38139034cdc9cd632f9dd1eab1a<br /><br />Tested successfully,
Thank you!!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36242">issues.joomla.org/tracker/joomla-cms/36242</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4404262d4cdaf56c0c7e5a5479dae7daca947e5d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36242">issues.joomla.org/tracker/joomla-cms/36242</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36242">issues.joomla.org/tracker/joomla-cms/36242</a>.</sub>

**Comment by bembelimen** [comment]
Thx, nice improvement.

**Pull request merged**